### ISSUE TITLE
problems installing cvmfs with ansibe

### ISSUE THREAD

**Issue by trustodia** [AAROC/CODE-RADE#124]
@brucellino 
```
ansible-playbook -c local -i localhost, cvmfs-client-2.2.yml -s --ask-sudo-pass

TASK [cvmfs-client-2.2 : download the CVMFS repo (RedHat)] *********************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: ImportError: No module named yum
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_o75WJ9/ansible_module_yum.py\", line 25, in <module>\n    import yum\nImportError: No module named yum\n", "module_stdout": "", "msg": "MODULE FAILURE"}
	to retry, use: --limit @/mnt/nfs/home/viz_phy/DevOps/Ansible/cvmfs-client-2.2.retry

PLAY RECAP *********************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1 
```

**Comment by trustodia** [comment]
@brucellino  ansible-playbook -c local -i localhost, cvmfs-client-2.2.yml -s --ask-sudo-pass
TASK [cvmfs-client-2.2 : Install CVMFS client] *********************************
skipping: [localhost] => (item=[]) 

TASK [cvmfs-client-2.2 : Add CVMFS repo] ***************************************
An exception occurred during task execution. To see the full traceback, use -vvv. The error was: , E:Some index files failed to download. They have been ignored, or old ones used instead.

ansible-playbook -c local -i localhost, cvmfs-client-2.2.yml -s --ask-sudo-pass --limit @/home/cubre/WACRE_HackFest/DevOps/Ansible/cvmfs-client-2.2.retry


TASK [cvmfs-client-2.2 : check that the repo is mounted] ***********************
failed: [localhost] (item=fastrepo.sagrid.ac.za) => {"changed": false, "failed": true, "failed_when_result": true, "item": "fastrepo.sagrid.ac.za", "stat": {"exists": false}}

**Comment by brucellino** [comment]
It looks like the error was network-related : 

```
The error was: , E:Some index files failed to download. They have been ignored, or old ones used instead.
```

**Comment by trustodia** [comment]
@brucellino this is on redhat

TASK [setup] *******************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "module_stderr": "Sorry, try again.\n[sudo via ansible, key=wljvtxfucsdnvgkqaxqxcygofdrymvtp] password: \nsudo: 1 incorrect password attempt\n", "module_stdout": "", "msg": "MODULE FAILURE"}
	to retry, use: --limit @/mnt/nfs/home/viz_phy/DevOps/Ansible/cvmfs-client-2.2.retry

PLAY RECAP *********************************************************************
localhost                  : ok=0    changed=0    unreachable=0    failed=1

**Comment by trustodia** [comment]
@brucellino 

> ansible-playbook -c local -i localhost, cvmfs-client-2.2.yml -s --ask-sudo-pass
TASK [cvmfs-client-2.2 : check that the repo is mounted] ***********************
failed: [localhost] (item=fastrepo.sagrid.ac.za) => {"changed": false, "failed": true, "failed_when_result": true, "item": "fastrepo.sagrid.ac.za", "stat": {"exists": false}}
	to retry, use: --limit @/home/cubre1/DevOps/Ansible/cvmfs-client-2.2.retry

PLAY RECAP *********************************************************************
localhost                  : ok=8    changed=1    unreachable=0    failed=1

**Comment by brucellino** [comment]
hi @trustodia 
Back from the holidays. We're doing quite a bit of work on the infrastructure at the moment, so I'm going to ask to put this on hold until 20-01-2017. The failure that you're seeing there is likely not fatal, but I'm not able to guarantee the availability of the repos during this period. Keep hacking, we'll be back soon... hang tight.

**Comment by brucellino** [comment]
closed in #123
